sdk/webpubsub/azure-messaging-webpubsubservice/samples/send_messages_connection_string_apim_proxy.py

```diff
@@ -0,0 +1,71 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+#
+# Copyright (c) Microsoft Corporation. All rights reserved.
+#
+# The MIT License (MIT)
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the ""Software""), to
+# deal in the Software without restriction, including without limitation the
+# rights to use, copy, modify, merge, publish, distribute, sublicense, and/or
+# sell copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+# FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
+# IN THE SOFTWARE.
+#
+# --------------------------------------------------------------------------
+import io
+import logging
+import os
+
+from azure.messaging.webpubsubservice import WebPubSubServiceClient
+from azure.core.exceptions import HttpResponseError
+
+logging.basicConfig(level=logging.DEBUG)
+LOG = logging.getLogger()
+
+try:
+    connection_string = os.environ['WEBPUBSUB_CONNECTION_STRING']
+    reverse_proxy_endpoint = os.environ["WEBPUBSUB_REVERSE_RPOXY_ENDPOINT"]
+except KeyError:
+    LOG.error("Missing environment variable 'WEBPUBSUB_CONNECTION_STRING' or 'WEBPUBSUB_REVERSE_RPOXY_ENDPOINT' - please set if before running the example")
+    exit()
+
+# Build a client from the connection string. And for this example, we have enabled debug
+# tracing. For production code, this should be turned off. 
+client = WebPubSubServiceClient.from_connection_string(connection_string, logging_enable=True, reverse_proxy_endpoint=reverse_proxy_endpoint)
```

Comment:
Also one thing I noticed in these samples: The actual calls you're doing with webpubsub are the same, the only difference is how to authenticate them. Do you think a better sample structure would be to have one sample with all of the different ways to authenticate (thinking something like [this](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/textanalytics/azure-ai-textanalytics/samples/sample_authentication.py), where we show multiple ways to authenticate), and then having one sample that is `client.send_to_all`, with just one way of authenticating (either DefaultAzureCredential or AzureKeyCredential, since those are the ones that most people will use)?

Reply:
Actually, I also think about that way. I hope customers could run sample file directly so the sample code should be simple asap. And for overall description about authentication, there is detailed info in `README.md`. So I hope to keep the sample like this format.